static/scripts/socket.js

```diff
@@ -1,5 +1,6 @@
 $(document).ready(function () {
-    let socket = io('http://localhost:3030');
+    let url = window.location.href.includes('localhost') ? 'http://localhost:3030' : 'https://schul-cloud.org:8080';
```

Comment:
I'm running http://127.0.0.1:3100/ locally ;)

Reply:
switched the cases, shouldn't matter anymore.